scripts/CameraJoint.gd

```diff
@@ -12,15 +12,28 @@ func _ready():
 
 func _physics_process(delta):
 	# zoom the camera in or out based on the scroll wheel
-	if Input.is_action_just_released("scroll_up"):
+	if Input.is_action_just_released("scroll_up") and spring_length > 0:
```

Comment:
A more readable version of this may be to set `zoom_length` based on the event status, including the sign, and then always add that to `spring_length` and subtract it from the `GeigerCounter` outside the if/else logic.

It ensures that the same value is used for all (relative) motions and only requires it to be calculated once.